drivers/power/supply/Kconfig

```diff
@@ -952,4 +952,11 @@ config CHARGER_QCOM_SMB2
 	  adds support for the SMB2 switch mode battery charger found
 	  in PMI8998 and related PMICs.
 
+config TSSILO_SUPERCAPS
+        tristate "embeddedTS SILO supercaps driver"
+        depends on I2C
+        select REGMAP_I2C
+        help
+          Only useful on boards with the Wizard device.
```

Comment:
Please expand this help to be more useful. What does this driver do? What platforms or situations should this be enabled on?